### DISCUSSION TITLE
How to annotate calculations

### DISCUSSION THREAD

**Discussion by tfinks** [wxMaxima-developers/wxmaxima#1552] [Q&A]
Is there a way to add text to identify each value nearby each assignment in a group of assignments like the following:

V_cc:12;    
V_DD:V_cc;
Q_GATE:172e-9;
I_LKGS:100e-9;
I_LKCAP:0;
I_QBS:150e-6;
I_LK:75e-6;
Q_LS:3e-9;
T_ON:2e-6;
I_LKDIODE:1e-6;

?

**Comment by tfinks** [comment]
My present solution is to split the (rather long) cell, and insert text cells to provide descriptions:

Driver supply voltage, nominal
(%i2)	V_cc:12$;  
	V_DD:V_cc$;
Total MOSFET gate charge
(%i3)	Q_GATE:172e-9$;
MOSFET gate-source leakage current
(%i4)	I_LKGS:100e-9$;
Bootstrap capacitor leakage current
(%i5)	I_LKCAP:0$;

**Comment by daute** [comment]
You mean to add comments? Yes, Maxima (and wxMaxima) allows C-style comments, e.g.:

```
V_cc:12;   /* V_cc means abc */
V_DD:V_cc; /* V_DD means def */
...
```

See: https://maxima.sourceforge.io/docs/manual/maxima_singlepage.html#Comments

**Reply by tfinks** [reply]
Ahh! That's helpful. Thank you.
